src/features/plans/panels/SearchPanel.tsx

```
@@ -0,0 +1,142 @@
+    <>
+      <div className="fixed inset-0 z-40 bg-background/60 backdrop-blur-sm" onClick={onClose} />
+
+      <aside className="fixed inset-y-0 right-0 z-50 w-full max-w-md border-l border-border bg-background shadow-xl">
+        <header className="flex items-center justify-between border-b border-border/50 p-4">
+          <h2 className="text-sm font-medium">장소 추가</h2>
+          <button
+            onClick={onClose}
+            className="flex h-8 w-8 items-center justify-center rounded-full hover:bg-secondary"
+          >
+            <X className="h-4 w-4" />
+          </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)